### PR TITLE
Update comms info, event info on site and README

### PR DESCRIPTION
**Pull request by keywordnew**
Fixes https://github.com/nodeschool/vancouver/issues/64

1. Update our README (https://github.com/nodeschool/vancouver) to show the response time and intended purpose (such as Github being for planning and meta issues) of each comm channel.
2. Update the meetup group with the same (not in diff :p )
3. Update the site (https://nodeschool.io/vancouver/) with the same.
4. While we're at it, update the site and README with a link to the new meetup group.
5. Make a few edits (see commits).

Site before:
screen shot 2017-12-09 at 02 42 35

Site after:
screen shot 2017-12-09 at 02 42 27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
 
-Check out our [events page](https://ti.to/nodeschool-vancouver). Feel
-free to come a little early and help set up!
+### Tier 1: Max response time within 1 day, real-time on workshop days
 
-### Want to mentor or help organize?
+- [twitter](https://twitter.com/nodeschoolyvr): tweet [@nodeschoolyvr](https://twitter.com/nodeschoolyvr) for general chat
+- [slack](https://yvrdev.slack.com): for general chat, planning meetups, and
+questions about workshops if you're challenging yourself at home. Signup [here](https://yvrdev.herokuapp.com/) and DM /@manil for an invite to our
+channel
```
Comment (Qard) [inline README.md]: Is there any particular reason not to just make the channel public?

Comment (keywordnew) [reply README.md]: I'm not very active on YVRDev. This way, it's a prompt to engage. But if it's better public from the start, that's fine.

Comment (Qard) [reply README.md]: I'm on yvrdev all the time, but I'm good either way.

**Review by Qard — APPROVED** [review]

**Comment by keywordnew** [comment]
Can we get this merged? 😄

**Comment by Qard** [comment]
Yeah, LGTM. Do you have merge powers?

**Comment by gyaresu** [comment]
Hey Stephen & Manil.

Figured I'd hijack the thread to suggest @chowdhurian requests to join the NodeSchool Org: https://github.com/orgs/nodeschool/dashboard

(I don't know if there's a link to share once you have submitted a request but if there is then Stephen and I can post to it)

And then you can be added as an admin to nodeschool/vancouver: https://github.com/nodeschool/vancouver/settings/collaboration

In other news: My new job has meant a shift back to Python and as such I've not been doing any Node practice.

So I just wanted to more explicitly say that while I probably won't be attending or organising events at the moment I am still happy to help with any github admin stuff.

Cheers, G.

( also ping @Qard )

**Comment by Qard** [comment]
You're always welcome to just stop by and hack on fun stuff. You don't need to be working with Node.js full-time to be welcome at NodeSchool! ❤️

**Comment by keywordnew** [comment]
Thanks Gareth! And congrats on the career moves too 🙌

I don't have merge powers, Stephen. I'll try what Gareth suggested. But I can't access the link ([dashboard](https://github.com/orgs/nodeschool/dashboard)).

I thought about opening an issue at [organizers](https://github.com/nodeschool/organizers/) but the theme of the issues on there don't seem to include requesting org access.

**Comment by gyaresu** [comment]
Thanks Manil :)

I've created an issue in the Nodeschool organisers. Folded in a suggestions about putting details on how to add members into the wiki for future travellers :)

https://github.com/nodeschool/organizers/issues/515

**Pull request merged**